README.md

```diff
@@ -36,86 +34,73 @@ pip install git+https://github.com/dynamiqs/dynamiqs.git
 
 ### Simulate a lossy quantum harmonic oscillator
 
-This first example shows simulation of a lossy harmonic oscillator with Hamiltonian $H=\omega a^\dagger a$ and a single jump operator $L=\sqrt{\kappa} a$ using QuTiP-defined objects:
+This first example shows simulation of a lossy harmonic oscillator with Hamiltonian $H=\omega a^\dagger a$ and a single jump operator $L=\sqrt{\kappa} a$.
 
 ```python
 import dynamiqs as dq
-import numpy as np
-import qutip as qt
-import torch
+import jax.numpy as jnp
 
 # parameters
-n = 128       # Hilbert space dimension
-omega = 1.0   # frequency
-kappa = 0.1   # decay rate
-alpha0 = 1.0  # initial coherent state amplitude
+n = 128      # Hilbert space dimension
+omega = 1.0  # frequency
+kappa = 0.1  # decay rate
+alpha = 1.0  # initial coherent state amplitude
 
-# QuTiP operators, initial state and saving times
-a = qt.destroy(n)
-H = omega * a.dag() * a
-jump_ops = [np.sqrt(kappa) * a]
-psi0 = qt.coherent(n, alpha0)
-tsave = np.linspace(0, 1.0, 101)
-
-# run on GPU if available, otherwise on CPU
-torch.set_default_device('cuda' if torch.cuda.is_available() else 'cpu')
+# initialize operators, state and saving times
+a = dq.destroy(n)
+H = omega * dq.dag(a) @ a
+jump_ops = [jnp.sqrt(kappa) * a]
+psi0 = dq.coherent(n, alpha)
+tsave = jnp.linspace(0, 1.0, 101)
 
 # run simulation
 result = dq.mesolve(H, jump_ops, psi0, tsave)
 print(result)
 ```
 
 ```text
-|██████████| 100.0% - time 00:00/00:00
 ==== Result ====
-Method       : Dopri5
-Start        : 2023-09-10 16:57:34
-End          : 2023-09-10 16:57:35
-Total time   : 0.48 s
```

Comment:
Note for later, we could add these informations again

Reply:
Indeed !